tests/committerparser.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+import sys
```

Comment:
Fixed.  Also added verified Py3 compatibility. :-)



From: Nicole Thomas <notifications@github.com>
Reply: saltstack/salt <reply@reply.github.com>>
Date: March 9, 2015 at 4:39:35 PM
To: saltstack/salt <salt@noreply.github.com>>
Cc: C. R. Oldham <cr@saltstack.com>>
Subject:  Re: [salt] Add simple parser to return some activity numbers from our git log. (#21455)  

In tests/committerparser.py:

> @@ -0,0 +1,104 @@
> +#!/usr/bin/python
> +import sys
This will need a from __future__ import absolute_path at the top here for python 3 compatibility.

—
Reply to this email directly or view it on GitHub.